packages/plugin-docusaurus-v3/src/index.ts

```diff
@@ -58,7 +58,7 @@ export default function OramaPluginDocusaurus(
     },
 
     async allContentLoaded({ actions, allContent }) {
-      const isDevelopment = !options.cloud?.oramaCloudAPIKey
+      const isDevelopment = process.env.NODE_ENV === 'development' && !options.cloud?.oramaCloudAPIKey
```

Comment:
Would you turn that into a `||` for backward compatibility? Thanks!

Reply:
No problem! Done! What about now?